docker.d/generate_version_json.sh

```diff
@@ -0,0 +1,17 @@
+#/bin/sh
+
+set -e
+test $PROJECT_NAME
+test $GIT_HEAD_REV
+test $TASK_ID
+
+version=$(cat version.txt)
```

Comment:
It looks like balrogscript uses `balrogscript/_version.py`. Since there is no `version.txt`, this is breaking the taskcluster graph. We should probably standardize how we specify \*script versions. In the monorepo, maybe we'll reach into the virtualenv or use `${PROJECT_NAME}_version.txt` or something.

Reply:
I added some potential fixes to https://github.com/escapewindow/balrogscript/commits/dockerization

Reply:
i will add 'version.txt` to all of the scriptworkers